### DISCUSSION TITLE
Ability to follow you as you map

### DISCUSSION THREAD

**Discussion by Swift8641** [streetcomplete/StreetComplete#4666] [Ideas]
It would be great if I could set a zoom level and have the app orient itself according to the direction I'm moving while keeping centered on the position indicator. OSM and and Google Maps can do this.

**Comment by FloEdelmann** [comment]
You can already do this in StreetComplete. Tap once on the cross hair button and it'll turn orange and keep your position centered on the map (while keeping the map oriented towards North). Tap the button again and it'll change to an arrow button and change to a 3D view which follows your position and moving direction.

**Reply by Swift8641** [reply]
Wow, I can't believe I missed that. Sorry guys.
